configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -820,7 +820,7 @@
                     <label>
                         <uimessage code="pihcore.mentalhealth.ThoughtContent"/>
                     </label>
-                    <obs conceptId="CIEL:167112" style="radio" answerConceptIds="PIH:NORMAL,PIH:ABNORMAL"  answerSeparator=""/>
+                    <obs conceptId="CIEL:167112" style="checkbox"  answerSeparator=""/>
```

Comment:
If you want checkboxes, you would add 1 obs (line) for each checkbox and include the answerConceptId.  For example, `<obs conceptId="CIEL:167112" style="checkbox" answerConceptId="CIEL:142600" /> <!-- Delusions checkbox -->`

Reply:
I do not understand @lnball . I text you on google Meet so we can discuss